### PR TITLE
Add MarginalSingle for Students T distribution

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

**Comment by btracey** [comment]
This also fixes some of the residual issues in #133 , and the declaration inconsistency with distuv/studentst

File: distmv/studentst.go [hunk 1]
```diff
@@ -284,9 +285,34 @@ func (s *StudentsT) MarginalStudentsT(vars []int, src *rand.Rand) (dist *Student
...
+	newSigma.SubsetSym(s.sigma, vars)
+	return NewStudentsT(newMean, &newSigma, s.nu, src)
+}
+
+// MarginalStudentsT returns the marginal distribution of the given input variable.
+// That is, MarginalStudentsT returns
+//  p(x_i) = \int_{x_o} p(x_i | x_o) p(x_o) dx_o
+// where i is the input index, and x_o are the remaining dimensions.
+// The input src is passed to the call to NewStudentsT.
+func (s *StudentsT) MarginalStudentsTSingle(i int, src *rand.Rand) distuv.StudentsT {
```
Comment (sbinet) [inline distmv/studentst.go]: perhaps add a link to https://en.wikipedia.org/wiki/Marginal_distribution ?

Comment (btracey) [inline distmv/studentst.go]: Done.

**Comment by btracey** [comment]
Ping

**Comment by sbinet** [comment]
LGTM (but probably somebody else should also have a look)

File: distmv/studentst.go [hunk 2]
```diff
@@ -284,9 +287,36 @@ func (s *StudentsT) MarginalStudentsT(vars []int, src *rand.Rand) (dist *Student
...
+	return NewStudentsT(newMean, &newSigma, s.nu, src)
+}
+
+// MarginalStudentsT returns the marginal distribution of the given input variable.
+// That is, MarginalStudentsT returns
+//  p(x_i) = \int_{x_o} p(x_i | x_o) p(x_o) dx_o
+// where i is the input index, and x_o are the remaining dimensions.
+// See https://en.wikipedia.org/wiki/Marginal_distribution for more information.
+//
+// The input src is passed to the call to NewStudentsT.
```
Comment (vladimir-ch) [inline distmv/studentst.go]: There is no call to `NewStudentsT`.

Comment (vladimir-ch) [inline distmv/studentst.go]: Sounds good.

**Pull request merged**